src/eggd_mosdepth.sh

```
@@ -57,38 +62,42 @@ main() {
-          --env bam_prefix="$bam_prefix" --env bam_file="$bam_file" \
+          --env input_prefix="$alignment_file_prefix" --env input_alignment_file="/data/$input_alignment_file" \
+          --env fasta="$fasta" \
           -w "/data/out/mosdepth_output" \
           $mosdepth_id /bin/bash -c \
-          'for label in $labels; do export $label; done; mosdepth $optional_arguments $bam_prefix $bam_file'  
+          'for label in $labels; do export $label; done; mosdepth $optional_arguments $fasta $input_prefix $input_alignment_file'
         else
-
         # no labels given, use default
+        echo $input_alignment_file
         sudo docker run -v `pwd`:/data -w "/data/out/mosdepth_output" $mosdepth_id /bin/bash -c \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)